_licenses/ofl-1.1.txt

```diff
@@ -1,33 +1,41 @@
 ---
 layout: license
-title: SIL Open Font License
+title: SIL Open Font License 1.1
 hidden: true
 permalink: /licenses/ofl/
+redirect_from: /licenses/ofl-1.1/
+filename: OFL.txt
 
 source: http://scripts.sil.org/OFL_web
 
-description: The Open Font License (OFL) is maintained by SIL International. It attempts to be a compromise between the values of the free software and typeface design communities. It is used for almost all open source font projects, including those by Adobe, Google and Mozilla.
+description: The Open Font License (OFL) is maintained by SIL International. It is designed to honor the values of both the Free Software and Typeface Design communities. It is used for many open font projects created by independent font developers, small foundries as well as big corporations like Adobe, Google and Mozilla.
```

Comment:
I'd rather the description focus on what benefit the license provides for font publishers and font consumers, rather than the history of the license or motivations of its creators. When would I use this license? Why would I use it?